exodus/reports/templates/report_details.html

```diff
@@ -5,7 +5,6 @@
 {% if report %}
   {% if request.user.is_superuser %}
     <p class="small alert alert-warning">
-      {% trans "Provide this URI to Electra for report completion:"%}
```

Comment:
If we remove this, should we remote the whole line? (with /api/report/xx)

Reply:
I thought it could be a good idea to get the URI suffix of the report.